packages/service-library/src/servicelib/logging_utils.py

```diff
@@ -121,6 +136,20 @@ def set_logging_handler(
         )
 
 
+class GeneralLogFilter(logging.Filter):
```

Comment:
`logging_utils` is becoming very big. What about moving this to `logging_filters.py`? Then rename as well the test etc...